### PR TITLE
[new profile] fix for single site affiliation bug

### LINKED ISSUE #5969: [22.0.0 - new profile] Empty drop down with only 1 site affiliation
**Describe the bug**
When a user entering a new candidate is only affiliated with 1 site, the site dropdown on the user profile page is empty

this bug was found by @thomasbeaudry

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
Fix issue where empty dropdown shows up in the module when the logged in user is only affiliated to 1 site.

I'm not a big fan of hiding fields with a single option, but judging by the code, that was the expected behaviour. So this is the smallest possible change to fix the bug on 22

#### Testing instructions (if applicable)

1. test items 5 and 6 of the testplan

#### Link(s) to related issue(s)

* Resolves #5969

### CODE REVIEW

File: modules/new_profile/test/TestPlan.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# New Profile Test Plan:
+
+1. Access New Profile page and ensure that it renders.
+   [Automation Testing]
+2. Access with useEDC on/off and ensure that the field appears/disappears
+   [Manual Testing]
```
Comment (PapillonMcGill) [inline modules/new_profile/test/TestPlan.md]: @ridz1208 Not a blocker but could you ensure that all phrases either end with a dot (preferably) or do not end with a dot?

Comment (ridz1208) [reply modules/new_profile/test/TestPlan.md]: @PapillonMcGill ma bad. Fixed it

**Comment by ridz1208** [comment]
@PapillonMcGill did you run make ?

the change is in the react file

**Comment by PapillonMcGill** [comment]
yup!

**Pull request merged**